platformifier/templates/upsun/.upsun/config.yaml

```diff
@@ -35,13 +35,14 @@ applications:
       {{ range $key, $value := .Mounts -}}
       "{{ $key }}":
         {{ range $key, $value := $value }}
+        {{- if eq $key "source" -}}{{- $value = $value | replace "local" "storage" -}}{{- end -}}
         {{- $key }}: "{{ $value }}"
         {{ end }}
       {{ end -}}
     {{- else -}}
     # mounts:
-    #   "/.cache": # Represents the path in the app.
-    #     source: "local" # "local" sources are unique to the app, while "service" sources can be shared among apps.
+    #   "/storage": # Represents the path in the app.
```

Comment:
can we use a different name for the _directory_ so it isn't confused with the `storage` source type?

Reply:
✅